corehq/apps/reports/static/reports/v2/js/datagrid/binding_handlers.js

```diff
@@ -171,4 +171,52 @@ hqDefine('reports/v2/js/datagrid/binding_handlers', [
         },
     };
 
+    ko.bindingHandlers.columnResize = {
```

Comment:
This is cool, I wonder where else it might be useful. Creating exports, maybe? Maybe the report modules / mobile UCR UI?

Reply:
should i aim to make this a global binding handler?